netcore/CMakeLists.txt

```diff
@@ -22,4 +22,7 @@ find_program(DOTNET dotnet)
 if(DOTNET AND INCLUDE_DOTNET)
   add_custom_target(netcore ALL COMMAND ${DOTNET} build --configuration $<CONFIG> WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/NetCore VERBATIM)
   add_custom_target(netcore-test ALL COMMAND ${DOTNET} build --configuration $<CONFIG> WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/NetCore.Test DEPENDS netcore VERBATIM)
+
+
+  install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/NetCore/bin/$<CONFIG>/netcoreapp3.1/Apache.Geode.Netcore.dll DESTINATION netcore/$<CONFIG>)
```

Comment:
Why are we including the 'Netcore' name in our library name? We don't include the language/platform name in any of our other library names.

Reply:
This is true; its a bit verbose, but does ensure no one will confuse the two .Net libraries... 
any suggestions on the naming?
`Apache.Geode.Platinum.dll`
`Apache.Geode.Modern.dll`

Reply:
Pretty hard to confuse the two since both compile time and runtime would fail. It seems common in the .NET space to have the assemblies and base package names consistent and to distinguish different versions of the platform by directory name.